doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_external_memory.rst

```diff
@@ -43,13 +41,53 @@ The companion image consists of two main parts:
 * IPC service exposed towards the Secure Domain
 
 When the companion image is booted and a directive that accesses the data on the external memory is issued, such as the ``suit-directive-fetch`` or ``suit-directive-copy`` directives, the Secure Domain firmware uses the IPC service provided by the companion image to access the contents of the external memory.
-Beyond the booting of the companion image, the update process does not differ from regular fetch model-based update.
+Beyond the booting of the companion image, the update process does not differ from regular push model-based or fetch model-based updates.
```

Comment:
```suggestion
After booting the companion image, the update process works the same as for regular push or fetch model updates.
```

Reply:
I'd rather change it to "apart from", not after, it is not about the sequence of operations.

Reply:
Changed to apart from - see if it's OK